compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/input/key/KeyEvent.web.kt

```diff
@@ -50,7 +50,7 @@ internal fun KeyboardEvent.toComposeEvent(): KeyEvent {
                 "keyup" -> KeyEventType.KeyUp
                 else -> KeyEventType.Unknown
             },
-            codePoint = key.codePointAt(0),
+            codePoint = if (key.firstOrNull()?.toString() == key) key.codePointAt(0) else 0,
```

Comment:
It's an incorrect check - the whole point of "codepoint" thing is to support multi-char symbols